recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -99,9 +99,14 @@ this.RecipeRunner = {
     }
   },
 
-  getFilterContext() {
+  getFilterContext(recipe) {
     return {
-      normandy: ClientEnvironment.getEnvironment(),
+      normandy: Object.assign(ClientEnvironment.getEnvironment(), {
```

Comment:
When using `Object.assign`, I tend to assign everything to an empty object to start - e.g. `Object.assign({}, ClientEnvironment.getEnvironment(), ...)`. Realistically, it probably doesn't make a difference.